src/Persistence/Reflection/EnumReflectionProperty.php

```diff
@@ -86,13 +86,25 @@ private function fromEnum($enum)
     }
 
     /**
-     * @param int|string|int[]|string[] $value
+     * @param int|string|int[]|string[]|BackedEnum|BackedEnum[] $value
      *
      * @return ($value is int|string ? BackedEnum : BackedEnum[])
```

Comment:
```suggestion
     * @return ($value is int|string|BackedEnum ? BackedEnum : BackedEnum[])
```

This should take down one SA baseline entry

Reply:
Didn't seem to, but good point anyway.